app/models/mail_server_log.rb

```diff
@@ -23,7 +23,7 @@ class MailServerLog < ActiveRecord::Base
 
   serialize :delivery_status, DeliveryStatusSerializer
 
-  before_create { |record| record.delivery_status }
```

Comment:
What was the error? They use this form in a [current example](http://api.rubyonrails.org/classes/ActiveRecord/Callbacks.html):

```ruby

class Firm < ActiveRecord::Base
  # Disables access to the system, for associated clients and people when the firm is destroyed
  before_destroy { |record| Person.where(firm_id: record.id).update_all(access: 'disabled')   }
  before_destroy { |record| Client.where(client_of: record.id).update_all(access: 'disabled') }
end
```